packages/realm-react/rollup.config.js

```diff
@@ -28,11 +29,16 @@ export default [
     output: [
       {
         file: pkg.main,
+        format: "cjs",
+        sourcemap: true,
+      },
+      {
+        file: pkg.module,
         format: "esm",
         sourcemap: true,
       },
     ],
-    plugins: [nodeResolve(), typescript({ noEmitOnError: true })],
+    plugins: [commonjs(), nodeResolve(), typescript({ noEmitOnError: true })],
```

Comment:
I'm curious if / why this was needed? What commonjs dependencies are loaded by `@realm/react`?

Reply:
Jest is commonjs by default.  It takes siginificant amount of manipulation to the jest framework to get it to support es modules.  
I have also found that many react native libraries also produce a cjs in their dist for this reason.

Reply:
Before this release, if you take a vanilla react native project...add `@realm/react` and add something using `@realm/react` in the default tests, it will fail on import

Reply:
Sure, but adding this line is not nessesary to emit a CommonJS bundle. It's only needed if you want to include some CommonJS dependency into the bundle.